test/tests/functional/pbs_qstat_formats.py

```diff
@@ -417,16 +417,17 @@ def test_qstat_json_valid_multiple_jobs_p(self):
         attribute with type resource list has to be the last attribute
         in order to hit the bug.
         """
-        a = {'resources_available.ncpus': 4}
-        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
+        self.server.manager(MGR_CMD_SET, NODE,
```

Comment:
why did you change ncpus from 4 to 3?

Reply:
This change is not required. I modified it to match the master.